public/modules/home/views/partials/footer.html

```diff
@@ -1,89 +1,102 @@
-    <div class="row">
-        <div class="col-xs-12 col-sm-4 col-md-4 col-lg-4">
-            <h3 class="footer-title">Updates</h3>
-            <div class="update">
-                <p>Mauris nec ultricies est, non rhoncus lacus. Donec elementum nunc luctus neque auctor, ut hendrerit ligula commodo. Ut vel sollicitudin nulla. In condimentum est eu vulputate dapibus</p>
-                <p><span>07/05/13</span><span class="pull-right">Admin</span></p>
-            </div>
-            <div class="update">
-                <p>Donec elementum nunc luctus neque auctor, ut hendrerit ligula commodo. Ut vel sollicitudin nulla. In condimentum est eu vulputate dapibus</p>
-                <p><span>24/04/13</span><span class="pull-right">Admin</span></p>
-            </div>
-        </div>
-        <div class="clearfix visible-xs"></div>
-        <div class="col-xs-12 col-sm-4 col-md-4 col-lg-4">
-            <h3 class="footer-title">Testimonials</h3>
-            <div class="testimonals">
-                <span class="picture"><img ng-src="modules/home/img/footer/testimonial-1.jpg" alt="" /> Peter Parker</span>
-                <p><i class="icon-quote-left"></i> Consectetur adipiscing elit. Sed ullamcorper ut lorem eu dapibus. In condimentum massa in dolor adipiscing tincidunt. Fusce mollis augue in libero tristique, a adipiscing felis condimentum. <i class="icon-quote-right"></i></p>
-                <div class="clearfix"></div>
-                <span class="picture"><img ng-src="modules/home/img/footer/testimonial-2.jpg" alt="" /> Joyce Kinney</span>
-                <p><i class="icon-quote-left"></i> In condi mentum massassa in dolor adipiscing tincidunt. Fusce augue in libero tristique, a adipiscing scing felis condimentum. <i class="icon-quote-right"></i></p>
-            </div>
-        </div>
-        <div class="clearfix visible-xs"></div>
-        <div class="col-xs-12 col-sm-4 col-md-4 col-lg-4">
-            <h3 class="footer-title">Contact</h3>
-            <form class="content-panel">
-                <div class="controls">
-                    <input id="name" name="name" type="text" class="input-block-level footer-input" placeholder="Name">
-                </div>
-                <div class="controls">
-                    <input id="email" name="email" type="email" class="input-block-level footer-input" placeholder="Email address">
-                </div>
-                <div class="controls">
-                    <textarea id="message" name="message" class="input-block-level footer-input" placeholder="Your Message" rows="5"></textarea>
-                </div>
-                <div class="controls">
-                    <button id="contact-submit" type="submit" class="btn btn-sky input-medium pull-right">Send</button>
-                </div>
-            </form>
-        </div>
+  <div class="row">
+    <div class="col-xs-12 col-sm-2 col-md-2 col-lg-2 col-sm-offset-1">
+      <h3 class="footer-title">Product</h3>
```

Comment:
Maybe the headers should be vertically aligned with their child items and start at the same x coordinate.